Source/Xamarin/Prism.Forms.Tests/Services/PageDialogServiceFixture.cs

```diff
@@ -20,68 +20,273 @@ public PageDialogServiceFixture()
         [Fact]
         public async Task DisplayActionSheetNoButtons_ShouldThrowException()
         {
-            var service = new PageDialogServiceMock("cancel", _applicationProvider);
-            var argumentException = await Assert.ThrowsAsync<ArgumentException>(() => service.DisplayActionSheetAsync(null, null));
-            Assert.Equal(typeof(ArgumentException), argumentException.GetType());
+            var service = new PageDialogServiceMock( "cancel", _applicationProvider );
+            var argumentException = await Assert.ThrowsAsync<ArgumentException>( () => service.DisplayActionSheetAsync( null, null ) );
+            Assert.Equal( typeof( ArgumentException ), argumentException.GetType() );
+        }
+
+        #region Tests using Action
```

Comment:
I put the regions there since we are deprecating the use of Commands, and keeping the grouping will make it easier to clean up when Commands are removed.